fe/fe-core/src/main/java/com/starrocks/planner/StreamLoadPlanner.java

```diff
@@ -154,6 +155,17 @@ public TExecPlanFragmentParams plan(TUniqueId loadId) throws UserException {
                 throw new DdlException("Only primary key table support partial update");
             }
         }
+        if (streamLoadInfo.getInsertMode() == TInsertMode.IGNORE_MODE) {
+            if (!isPrimaryKey) {
+                throw new DdlException("Only primary key table support ignore insert mode");
+            }
+            if (streamLoadInfo.isPartialUpdate()) {
+                throw new DdlException("Ignore insert mode is not support partial update.");
+            }
+            if (streamLoadInfo.getMergeConditionStr() != null && !streamLoadInfo.getMergeConditionStr().isEmpty()) {
+                throw new DdlException("Ignore insert mode is not support condition update.");
+            }
```

Comment:
ignore insert mode -> insert ignore mode ?
is not support -> does not support
`is not support condition update` -> `does not support conditional update`

Reply:
fine